src/gatsby/utils/resolveOpenAPI.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
+import { promises as fs } from "fs";
+
+const activeEnv =
+  process.env.GATSBY_ENV || process.env.NODE_ENV || "development";
+
+export default async () => {
+  if (activeEnv === "development" && process.env.OPENAPI_LOCAL_PATH) {
+    try {
+      console.log(`Fetching from ${process.env.OPENAPI_LOCAL_PATH}`);
+      let data = await fs.readFile(process.env.OPENAPI_LOCAL_PATH, "utf8");
+      return data;
+    } catch (error) {
+      console.log(
+        `Failed to connect to  ${process.env.OPENAPI_LOCAL_PATH}. Continuing to fetch versioned schema from Github.
+        ${error}`
+      );
+    }
+  }
```

Comment:
I think if you wanted you could add another `console.log` indicating that you are fetching from the `sentry-api-schema`. I forgot to put `export` in front of the `OPENAPI_LOCAL_PATH` and was confused for a min about why I didn't see the console.log message.

Reply:
@MeredithAnya hmm should we log defaults? I don't think we currently do that for anything else.